### DISCUSSION TITLE
Conda installation is stuck on "Solving environment"

### DISCUSSION THREAD

**Discussion by mfkeles** [talmolab/sleap#1465] [Help!] [answered]
I am curious if anyone has run into this issue. I had sleap already installed and working but wanted to update to the latest. I was never able to achieve it. I deleted conda and did a fresh install but still the same issue. I am just running: 

`conda create -y -n sleap -c conda-forge -c nvidia -c sleap -c anaconda sleap`

I noticed that as this is running and stuck on stuck solving environment the memory usage increases significantly and constantly goes up. 

I am able to create a new environment and install other packages.

**Comment by mfkeles** [comment, marked as answer]
For those of you running similar issue, using libmamba as the config solver fixed it

https://www.anaconda.com/blog/a-faster-conda-for-a-growing-community

**Reply by roomrys** [reply]
Hi @mfkeles,

For any installation of [SLEAP v1.3.1](https://github.com/talmolab/sleap/releases/tag/v1.3.1)+ we recommend [mamba](https://mamba.readthedocs.io/en/latest/installation.html) (or micromamba) instead of miniconda for package resolution. If you have an existing miniconda installation, then setting the solver to libmamba should also work (I'll include this in the release notes).

This is because we replaced many of the PyPI distributed dependencies to Anaconda distributed (to increase compatibility across OS's). The downside is that more Anaconda dependencies results in slower package resolution. Hence the need for mamba.

Thanks,
Liezl
